webpack/index.d.ts

```diff
@@ -654,7 +654,20 @@ declare namespace webpack {
      */
 
     class BannerPlugin extends Plugin {
-        constructor(banner: any, options: any);
+        constructor(options?: BannerPlugin.Options | string);
+    }
+
+    namespace BannerPlugin {
+        type Filter = string | RegExp;
+        
```

Comment:
remove whitespace; you can run `npm run lint -- webpack` and `npm test` to test it locally